app/views/devise/sessions/new.html.erb

```diff
@@ -5,7 +5,7 @@
 
 <h2>Log in</h2>
 
-<%= form_for(resource, as: resource_name, url: session_path(resource_name), class: "session") do |f| %>
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
   <div class="field">
     <%= f.label :email, class: "form-label" %><br />
     <%= f.email_field :email, autofocus: true, autocomplete: "email",
```

Comment:
- [ Required ] I think it will be much better if you made it possible for users to create accounts from the `http://127.0.0.1:3000/users/sign_up` page, currently, when I try to create an account on your application I get the error below. Please see the screenshot below and rectify this.

- [ Required ] Once this error is fixed please ensure that the design of the page matches the wireframe provided for the sign In page by Microverse for this project, currently, I can't confirm this because the page has an error.

## Error in signup view
![Screenshot from 2023-05-25 00-25-37](https://github.com/juaryimami/blog-app/assets/16392046/e9665e04-5958-4e1f-84a6-9504973d2ae9)

- [ Required ] I think it will be much better if you made it possible for users to login into accounts from the `http://127.0.0.1:3000/users/sign_in` page, currently when I try to login into the application I get the error below. Please see the screenshot below and rectify this.

- [ Required ] Once this error is fixed please ensure that the design of the page matches the wireframe provided for the sign In page by Microverse for this project, currently, I can't confirm this because the page has an error.

## Error in sign-in view
![Screenshot from 2023-05-25 00-26-06](https://github.com/juaryimami/blog-app/assets/16392046/37a1b169-7619-4577-bb23-0f1fed49078d)